europace_security.yaml

```
@@ -14,99 +14,107 @@ flows:
-        ## Kreditsmartvorgänge schreiben
-        Der Client kann KreditSmart-Vorgänge automatisiert anlegen.
+        ## KreditSmart-Vorgänge anlegen
 
       unterlagen:dokument:lesen: |
-        ## Dokumente lesen
-        Der Client kann hochgeladene Dokumente eines Vorgangs abrufen.
+        ## Unterlagen-Dokumente herunterladen
+        Hochgeladene Dokumente eines Vorgangs können heruntergeladen werden.
       unterlagen:dokument:schreiben: |
-        ## Dokumente schreiben und kategorisieren
-        Der Client kann Dokumente zu einem Vorgang hochladen, umbenennen, löschen und die Kategorisierung anstoßen, damit die Dokumente in der Unterlagenakte zu Verfügung stehen.
+        ## Unterlagen-Dokumente hinzufügen und kategorisieren
+        Dokumente zu einem Vorgang können hochladen, umbenannt, gelöscht und die Kategorisierung gestartet werden, damit die Dokumente in der Unterlagenakte zu Verfügung stehen.
       unterlagen:unterlage:lesen: |
         ## Unterlagen lesen
-        Der Client kann die kategorisierten Dokumente (Unterlagen) abrufen.
+        Kategorisierten Dokumente (Unterlagen) können abgerufen werden.
       unterlagen:unterlage:schreiben: |
         ## Unterlagen neu zuordnen
-        Der Client kann die Kategorie und den Bezug der Unterlagen ändern.
+        Die Kategorie und der Bezug der Unterlagen kann geändert werden.
       unterlagen:unterlage:freigeben: |
         ## Unterlagen freigeben
-        Der Client kann Unterlagen für einen Antrag freigeben.
+        Unterlagen für einen Antrag können freigeben werden.
       unterlagen:freigabe:lesen: |
         ## Freigegebene Unterlagen lesen
-        Der Client kann freigegebene Unterlagen zu einem Antrag abrufen
+        Freigegebene Unterlagen zu einem Antrag können abgerufen werden.
       unterlagen:freigabe:schreiben: |
         ## Freigegebene Unterlagen aktualisieren
-        Der Client kann den Status einer freigegeben Unterlagen setzen, um aus Produktanbietersicht den Empfang der Unterlagen zu bestätigen.
+        Der Status einer freigegeben Unterlagen kann verändert werden, um aus Produktanbietersicht den Empfang der Unterlagen zu bestätigen.
 
       impersonieren: |
-        ## Darf Impersonieren
-        Eine Aktion wird im Namen eines Partners ausgeführt wobei die Autorisierung des Clients durch einen übergeordneten Partner erfolgt.
+        ## Impersonieren
+        Aktionen können im Namen eines untergeordneten Partners ausgeführt werden.
       openid: |
-        ## Benutzer-Login durchführen
-        Der Client kann das Login des Benutzers auf der Plattform mittels OpenID Connect anfordern und die Identität des Benutzers abrufen.
+        ## Benutzer anmelden
+        Der Benutzers kann automatisch angemeldet werden.
       profile: |
-        ## Benutzerprofil lesen
-        Der Client kann zusätzlich zur Identität (Partner ID) Profildaten des eingeloggten Benutzers abrufen.
+        ## Benutzer-Profil lesen
```

```suggestion
        ## Benutzeprofil lesen
```
Das muss meiner Meinung nach zusammengeschrieben werden, das ist ein etabliertes Wort.